engine/hatchery/serve.go

```diff
@@ -187,6 +190,46 @@ func (c *Common) getPanicDumpListHandler() service.Handler {
 	}
 }
 
+func (c *Common) InitServiceLogger() error {
+	tcpServer := c.Common.ServiceInstance.LogServer
+	var signer jose.Signer
+	if tcpServer.Addr != "" && tcpServer.Port != 0 {
+		logger, err := log.New(fmt.Sprintf("%s:%d", tcpServer.Addr, tcpServer.Port))
+		if err != nil {
+			return sdk.WithStack(err)
+		}
+		signer, err = jws.NewSigner(c.Common.PrivateKey)
+		if err != nil {
+			return sdk.WithStack(err)
+		}
+		c.Signer = signer
+		c.ServiceLogger = logger
+	}
```

Comment:
if we are unable to start a logger (because the conf is empty), we should raise an error

Reply:
If conf is empty, it use the previous system. I want to keep this, to be able to rollback the feature by changing the configuration

Reply:
or enable it by configuration